plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/PrepareCodeGenerationState.kt

```diff
@@ -49,10 +50,11 @@ class PrepareCodeGenerationState(
             )
 
             uploadArtifactToS3(uploadUrlResponse.uploadUrl(), fileToUpload, zipFileChecksum, zipFileLength, uploadUrlResponse.kmsKeyArn())
+            deleteUploadArtifact(fileToUpload)
```

Comment:
Why have we added delete here as well ?

Reply:
We didn't add delete here, just moved it from below (line#64) to here to have file-related actions next to each other before starting to init next state.